content/docs/en/utilities/v-template.md

```diff
@@ -1,57 +1,46 @@
 ---
-title: v-template component
-contributors: [rigor789]
+title: v-template
+contributors: [rigor789, ikoevska]
 ---
 
-The `v-template` component is used for defining markup that can be reused as a template.
-
-It is utilized internally by the [ListView Component](/en/docs/elements/components/list-view).
+The `<v-template>` component lets you define markup that you can reuse as a template.
 
 ## Props
 
-| name | type | description |
+| Name | Type | Description |
 |------|------|-------------|
 | `if` | `String` | The condition for using this template.
 | `name` | `String` | The name of the template, auto-generated if omitted.
 
-# Advanced usage
-
-The rest of this document is meant for advanced users implementing their own components that require a template or multiple templates.
-
-`v-template` does not render anything when placed into the template, instead it adds a `$templates` property to the parent element or component, which is a [`TemplateBag`](https://github.com/nativescript-vue/nativescript-vue/blob/master/platform/nativescript/runtime/components/v-template.js#L36) instance, and then registers itself as an available template.
-
-### TemplateBag
-
-The `TemplateBag` class is useful for registering multiple templates, and provides a function for selecting the correct template based on the item and the conditions provided for each template.
+## Basic usage
 
-Templates are stored as objects conforming to the [`KeyedTemplate`](https://docs.nativescript.org/api-reference/interfaces/_ui_core_view_.keyedtemplate) interface.
-
-#### The `selectorFn` property
+The `<v-template>` component is used internally by the [ListView component](/en/docs/elements/components/list-view) to iterate over its list items.
 
-The `selectorFn` property returns a function that accepts a single parameter which should be an item whose template should be selected. It will go through all the templates registered in the `TemplateBag` and return the first one where the `if` condition returns a truthy value, and if none of the templates match it will return `default`.
+## Advanced usage
 
-#### Available Methods
+You can use `v-template` to implement custom components that require a template or multiple templates.
 
-##### registerTemplate(name: String, condition: String?, scopedFn: Function): void
+`v-template` does not render anything when placed in the template. Instead, it adds a `$templates` property registers itself as an available template in the parent element. The parent element is a [`TemplateBag` instance](https://github.com/nativescript-vue/nativescript-vue/blob/master/platform/nativescript/runtime/components/v-template.js#L36).
```

Comment:
This is saying that the parent element is a `TemplateBag`, but what it should say is that `v-template` adds a property to the parent element called `$templates` and this property is a `TemplateBag` instance - and then registers itself as an available template in this TemplateBag (which is reused if it already exists on the parent)